src/hardware/pcspeaker.cpp

```diff
@@ -214,51 +219,86 @@ void PCSPEAKER_SetCounter(Bitu cntr,Bitu mode) {
 		return;
 	}
 	spkr.pit_mode=mode;
+
+	// Activate the channel after queuing new speaker entries
+	spkr.chan->Enable(true);
+}
+
+// Returns the neutral voltage if the speaker's  fully faded,
+// otherwise returns the fallback if the speaker is active.
+float NeutralOr(float fallback)
+{
+	return !spkr.fade_step ? SPKR_NEUTRAL_VOLTAGE : fallback;
 }
 
+// Returns, in order of preference:
+// - Neutral voltage, if the speaker's fully faded
+// - The last active PIT voltage to stitch on-going playback
+// - The fallback voltage to kick start a new sound pattern 
+float NeutralLastPitOr(float fallback)
+{
+	return NeutralOr(std::isgreater(fabs(spkr.pit_last), SPKR_NEUTRAL_VOLTAGE)
```

Comment:
Maybe? (I find it  a bit more readable)
```suggestion
	const bool use_last = std::isgreater(fabs(spkr.pit_last),
	                                     SPKR_NEUTRAL_VOLTAGE);
	return NeutralOr(use_last ? spkr.pit_last : fallback);
```

Reply:
The `Neutral` part has a specific meaning and implies that's the default or preferred values, and that the supplied argument is the fallback, or the `Maybe` value.

It is tricky to name them though.. will go with `NeutralOrMaybe(...)`, which I think is even better.

Reply:
Ah, I was not actually suggesting the function name change, only implementation change :)

I am fine with `NeutralOr` (many Rust and some Java functions follow similar pattern in naming - I think it's good signifier for passed value being a fallback).

Reply:
Oh ok; yeah, I'll flip the function names back.

> I was not actually suggesting the function name change, only implementation change

Need more help in which variable  :-)
- `const bool use_last = ` switch to `const bool maybe = `, or
- rename `fallback` to `maybe`?

Reply:
I think I was too imprecise in my review comment :) I do not suggest using word Maybe here at all; I am suggesting replacing the implementation of `NeutralLastPitOr` with:
```
	const bool use_last = std::isgreater(fabs(spkr.pit_last),
	                                     SPKR_NEUTRAL_VOLTAGE);
	return NeutralOr(use_last ? spkr.pit_last : fallback);
```

Reply:
Lol!  After seeing you had used 'Maybe' for a function in the past, I really got hung up on that here 🤣. 
I've reverted the names, and dropped in your improved suggestion: way more readable! Thank you.